Orbitersdk/samples/ProjectApollo/src_csm/saturn.cpp

```diff
@@ -4750,6 +4757,97 @@ void Saturn::StartSeparationPyros()
 	payloadCommandConnector.StartSeparationPyros();
 }
 
+void Saturn::UpdateMassAndCoG()
+{
+	CurrentFuelWeight = 0;
+	if (ph_sps != NULL) { CurrentFuelWeight += GetPropellantMass(ph_sps); }
+	if ((LastFuelWeight - CurrentFuelWeight) > 100.0) {
+		// Update physical parameters
+		VECTOR3 pmi, CoG;
+		CalculatePMIandCOG(pmi, CoG);
+		// Use SetPMI, ShiftCG, etc.
+		VECTOR3 CoGShift = CoG - currentCoG;
+		ShiftCG(CoGShift);
+		SetPMI(pmi);
+		currentCoG = CoG;
+
+		//Touchdown Points
+		//DefineTouchdownPoints(stage);
+
+		//Lights
+
+		// All done!
+		LastFuelWeight = CurrentFuelWeight;
+	}
+}
+
+void Saturn::CalculatePMIandCOG(VECTOR3 &PMI, VECTOR3 &COG)
+{
+	//Empty SM including SM RCS prop and SPS residuals
+	static const double MSM = 10675.0;
+	static const VECTOR3 CGSM = _V(918.7, -5.7, 11.2);
+	//CM including CM RCS
+	static const double MCM = 12392.0;
+	static const VECTOR3 CGCM = _V(1042.1, -0.2, 5.8);
+	//Sump tank capacity
+	static const double MSUMP = 22300;
+
+	double propmass = CurrentFuelWeight / 0.453597;
+
+	double mass, sumpm, storem, oxstorem, fuelstorem, oxsumpm, fuelsumpm;
+	
+	mass = propmass + MCM + MSM;
+
+	if (propmass > MSUMP)
+	{
+		sumpm = MSUMP;
+		storem = propmass - MSUMP;
+	}
+	else
+	{
+		sumpm = propmass;
+		storem = 0;
+	}
```

Comment:
Let's be consistent with the way we put braces

```suggestion
	if (propmass > MSUMP) {
		sumpm = MSUMP;
		storem = propmass - MSUMP;
	} else {
		sumpm = propmass;
		storem = 0;
	}
```